packages/react-three-rapier/src/utils/utils.ts

```diff
@@ -99,3 +100,13 @@ export function useConst<T>(initialValue: T | (() => T)): T {
   }
   return ref.current.value;
 }
+
+export const vector3ToRapierVector = (v: Vector3ish) => {
+  if (Array.isArray(v)) {
+    return new RapierVector3(v[0], v[1], v[2]);
+  } else if (typeof v === "number") {
+    return new RapierVector3(v, v, v);
+  } else {
+    return v as THREE.Vector3;
```

Comment:
It's probably best to always construct a rapier Vector3 for interacting with the rapier api.
While this may work now (haven't tested), rapier could make changes in future that break this.

Reply:
Ya good call - it also did look odd to the return either a THREE.Vector3 or Rapier Vector. This is likely the right approach. 👍